react/tic-tac-toe/src/app/screens/Login/styles.module.scss

```diff
@@ -0,0 +1,66 @@
+.field {
+  margin-bottom: 16px;
+}
+
+.label {
+  color: #4A5568;
+  display: block;
```

Comment:
Is this necessary?

Reply:
It's not. I think it's better remove it.